### PR TITLE
Feat: cart management page 

### PR DESCRIPTION
**Pull request by vivekmehto**
image

### CODE REVIEW

File: routes/cart.html [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+                <button class="btn btn-grey btn-block">Remove from cart</button>
+                <button class="btn btn-grey btn-block">Move to wishlist</button>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div class="cart-price flex flex-col gap p-01 b-rad-m">
+          <ul class="list">
+            <div class="head-3">Categories</div>
```
Comment (SimranjitKaurMaan) [inline routes/cart.html]: This shouldn't be categories. Should be corrected.

**Review by SimranjitKaurMaan — APPROVED** [review]
Looks good to me. One typo but can be merged.

File: routes/cart.html (hunk 1 above)
Comment (vivekmehto) [reply routes/cart.html]: Yes forgot to change.thank you

**Review by theshivamverma — APPROVED** [review]
Good use of reusable classes, good to merge.

**Pull request merged**